src/modules/Dashboard/features/Respondents/Respondents.tsx

```diff
@@ -415,7 +416,28 @@ export const Respondents = () => {
         return t('noMatchWasFound', { searchValue });
       }
 
-      return appletId ? t('noRespondentsForApplet') : t('noRespondents');
+      return appletId ? (
```

Comment:
Could you please move the renderEmptyComponent function to its own component, such as Respondents/EmptyState? This could help separate concerns and improve code organization.